applications/gfs_init.py

```diff
@@ -0,0 +1,36 @@
+from credit.nwp import build_GFS_init, format_datetime
+from credit import metadata
+import yaml
+import argparse
+import xarray as xr
+import os
+from os.path import join
+import pandas as pd
+
+username = os.environ.get('USER')
+base_path = '/glade/work/username/'.replace("username", username)
+base_out_path = '/glade/derecho/scratch/username/CREDIT/GFS_INIT/'.replace("username", username)
```

Comment:
Please do not hard code GLADE paths into the script. We would like to use this on laptops, cloud VMs, and other machines in general. Make these arguments in argparse.

Reply:
Do you prefer these be added as a separate arg flag on added to the base config?

Reply:
Put it in the config.

Reply:
Added tiny new conf file and cleaned paths. Ready for re-review